### DISCUSSION TITLE
Retrieving data from WebHook as Update

### DISCUSSION THREAD

**Discussion by aizekjohn** [westacks/telebot#92] [Q&A] [answered]
It's been a long that I haven't used TeleBot with Laravel, but I've had experience with this library.
With version 1.12 after setting webhook up, I've used
`$update = TeleBot::handleUpdate();` this to retrieve Update via webhook.
I didn't have any pre-added handlers, I used the $update so I can callHandler situationally.
Now with the latest version the code isn't working, can anyone help me ?

**Comment by punyflash** [comment, marked as answer]
Since 2.0 you need manually create an update object:
```php
$update = new Update($request->post())
```
